admin/class-help-center.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class WPSEO_Help_Center
+ */
+class WPSEO_Help_Center {
+	/**
+	 * @var array Help_Center_Item list of helpcenter tabs to be shown in the helpcenter
+	 */
+	private $help_center_items = array();
+
+	/**
+	 * @var String $group_name
+	 */
+	private $group_name;
+
+	/**
+	 * @var WPSEO_Option_Tab $tab
+	 */
+	private $tab;
+
+	/**
+	 * WPSEO_Help_Center constructor.
+	 *
+	 * @param String           $group_name The name of the group of the tab the helpcenter is on.
+	 * @param WPSEO_Option_Tab $tab        The name of the tab the helpcenter is on.
+	 */
+	function __construct( $group_name, $tab ) {
+		$this->group_name = $group_name;
+		$this->tab        = $tab;
+
+		$kb_help_center_item = new WPSEO_Help_Center_Item(
+			'knowledge-base',
+			__( 'Knowledge base', 'wordpress-seo' ),
+			array(
+				'content'        => '<div class="wpseo-kb-search"></div>',
+				'view_arguments' => array( 'identifier' => $tab->get_name() ),
+			)
+		);
+		array_push( $this->help_center_items, $kb_help_center_item );
+
+		$popup_title = sprintf( __( 'Email support is a %s feature', 'wordpress-seo' ), 'Yoast SEO Premium' );
+		/* translators: %1$s: expands to 'Yoast SEO Premium', %2$s: links to Yoast SEO Premium plugin page. */
+		$popup_content = sprintf( __( 'To be able to contact our support team, you need %1$s. You can buy the plugin, including one year of support, updates and upgrades, on %2$s.', 'wordpress-seo' ),
+			'<a href="https://yoast.com/wordpress/plugins/seo-premium/#utm_source=wordpress-seo-metabox&utm_medium=popup&utm_campaign=multiple-keywords">Yoast SEO Premium</a>',
+			'yoast.com' );
+
+		$premium_popup = new WPSEO_Premium_Popup( 'contact-support', $popup_title, $popup_content );
+		$contact_support_help_center_item = new WPSEO_Help_Center_Item(
+			'contact-support',
+			__( 'Email support', 'wordpress-seo' ),
+			array( 'content' => $premium_popup->get_premium_message( false ) )
+		);
+
+		array_push( $this->help_center_items, $contact_support_help_center_item );
+	}
+
+	/**
+	 * @return mixed (WPSEO_Help_Center_Item | null) Returns a help center item containing a video.
+	 */
+	private function get_video_help_center_item() {
+		$url = $this->tab->get_video_url();
+		if ( empty( $url ) ) {
+			return null;
+		}
+
+		return new WPSEO_Help_Center_Item(
+			'video',
+			'Video tutorial',
+			array(
+				'view'           => 'partial-help-center-video',
+				'view_arguments' => array(
+					'tab_video_url' => $url,
+				),
+			)
+		);
+	}
+
+	/**
+	 * Outputs the help center.
+	 */
+	public function draw_help_center() {
+		array_unshift( $this->help_center_items, $this->get_video_help_center_item() );
```

Comment:
I would put the line 86-88 in its own method, now this method has two responsibilities.